project/Version.scala

```diff
@@ -4,7 +4,8 @@ object Version {
   def either(environmentVariable: String, default: String): String =
     Properties.envOrElse(environmentVariable, default)
 
-  val floodmodel = either("TRAVIS_TAG", "SNAPSHOT")
+  val version = "0.1.3"
+  val floodmodel = version + either("FLOODMODEL_VERSION_SUFFIX", "-SNAPSHOT")
```

Comment:
It's supposed to be an optional environment variable. Currently, it's not being used. I'm not sure why it was included in the original commit https://github.com/azavea/usace-flood-geoprocessing/blob/bf1724b934b3b17391d23c7c9704b84f321aa431/project/Version.scala